packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -1086,9 +1087,13 @@ export const formNestedStack = async (
   let categories = Object.keys(amplifyMeta);
 
   categories = categories.filter((category) => category !== 'providers');
-  categories.forEach((category) => {
+
+  for (const catIdx in categories) {
+    const category = categories[catIdx];
```

Comment:
nit: I think this code can be simplified a bit by using `Object.values` or `Object.entries` on line 1087 and 1093, but not a blocker

Reply:
I need to refactor this to use for/of